main.go

```diff
@@ -92,7 +92,7 @@ Options:
   --disable-events                           Disable events collecting and sending(Deprecated).
   --disable-scalar                           Disable in-agent scalar.
   --port <port>                              Port to start the server on for liveness and readiness probes
-                                               [default: 80]
+                                               [default: 8080]
```

Comment:
This should not be changes as it will cause problems with already created clusters. Instead add an argument to the deployment to set the port to `8080`